theseus/embodied/kinematics/robot_model.py

```diff
@@ -39,3 +41,27 @@ def forward_kinematics(self, robot_pose: LieGroup) -> torch.Tensor:
 
     def dim(self) -> int:
         return 1
+
+class UrdfRobotModel(RobotModel):
+    def __init__(self, urdf_path: str):
+        self.robot_model = drm.DifferentiableRobotModel(urdf_path)
+    
+    def forward_kinematics(self, joint_states: torch.Tensor) -> Dict[str, SE3]:
+        """Computes forward kinematics
+        Args:
+            joint_states: Vector of all joint angles
+        Outputs:
+             Dictionary that maps link name to link pose
+
+        (Note: differentiable robot model uses the xyzw quaternion convention, while theseus uses wxyz, hence the conversion.)
+        """
+        link_poses = {}
+        for link_name in self.robot_model.get_link_names():
+            pos, quat = self.robot_model.compute_forward_kinematics(joint_states, link_name)
```

Comment:
Note that for consistency we will be using rotation then translation everywhere #76, so this will get modified a bit. cc @luisenp @fantaosha